nixos/modules/services/networking/syncthing.nix

```diff
@@ -335,10 +337,21 @@ in {
                   upstream's docs</link>.
                 '';
               };
-
             };
           }));
         };
+
+        extraOptions = mkOption {
+          type = types.attrs;
```

Comment:
maybe something like `(pkgs.formats.json {}).type` would be cooler? so stuff can be defined at multiple locations?

Reply:
Done.